dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -126,7 +152,7 @@ const SavedRunsComponent = () => {
               </Tr>
             </Thead>
             <Tbody>
-              {savedRuns.map((item, rowIndex) => {
+              {initSavedRuns.map((item, rowIndex) => {
```

Comment:
Isn't `initSavedRuns` always an empty array?  (I don't see anywhere where it is changed.)

Also, why do we use `savedRuns.length` instead of `initSavedRuns.length` at line 189 if we're using `initSavedRuns` here?

Reply:
`initSavedRuns` is being set in in `initalizeRuns()`

'savedRuns.length' is used to get the total number of  saved runs.